R/build.r

```diff
@@ -131,6 +131,7 @@
 #' of the components, and whether they're right aligned or left aligned.
 #' You can use this component to change the order of the default components,
 #' and to add your own components.
+#' Any unplaced components go to the right of the left navbar.
```

Comment:
this comes from a code comment + https://github.com/r-lib/pkgdown/issues/1354#issuecomment-655677645